src/libraries/Microsoft.PowerFx.Core/Public/Config/ReadOnlySymbolTable.cs

```diff
@@ -348,8 +348,9 @@ bool INameResolver.LookupEnumValueByTypeAndLocName(DType enumType, DName locName
 
         // Methods from INameResolver that we default / don't implement
         IExternalDocument INameResolver.Document => default;
-
-        IExternalEntityScope INameResolver.EntityScope => default;
+
+        internal virtual IExternalEntityScope InternalEntityScope => default;
```

Comment:
>InternalEntityScope [](http://example.com/codeflow?start=46&length=19)

Why do we need this? 

PowerApps bypasses the SymbolTable classes and uses INameResolver directly.

Reply:
This is needed for the unit tests, since the delegation validation logic is still using the SymbolTable